### DISCUSSION TITLE
update process: post-move?

### DISCUSSION THREAD

**Discussion by javierbarellano4v** [matter-js/matter.js#66] [Q&A] [answered]
With the move to Project CHIP, I'd like to confirm the update process.

Are these still the appropriate steps to update to the latest:

1. Get the sources git clone https://github.com/project-chip/matter.js.git 
2. cd node-matter
3. Install the dependencies: npm i
4. Build: npm run build
5. Run: node build/Device.js
6. (optional) Install globally: npm -g i . and run it with matter

***note, git repo has been modified to repo under project-chip**

**Comment by Apollon77** [comment, marked as answer]
it is in fact no "update". You start over anew with matter.js. ... the relevant code from node-matter is moved into matter.js/packages/matter-node.js

I adjusted your steps:
1. Get the sources git clone https://github.com/project-chip/matter.js
2. cd matter.js
3. Install the dependencies (execute in the matter.js root dir, for all packages, will also automatically build it): npm i
4. Run: node packages/matter-node.js/dist/Device.js

For global install honestly I never tried that ...

**Comment by javierbarellano4v** [comment]
@Apollon77 Thanks.
